search-app/src/cmr/search/system.clj

```diff
@@ -55,7 +79,11 @@
   "App logging level"
   {:default "info"})
 
-(def search-public-conf
+(def public-conf
```

Comment:
Oh, wait ... this is search, not ingest. I'm confused too :-) Regardless, I've updated all of them to be functions :-)